package.json

```diff
@@ -28,7 +29,7 @@
     "typings": "^1.3.2"
   },
   "scripts": {
-    "test": "npm run format && tsc && tslint -c tslint.json src/*.ts src/**/*.ts ; mocha",
```

Comment:
clang-format branch merged-- i think this was removed because formatting was being lame-- hopefully new formatting doesn't cause problems and we can keep aggressively clang-format'n